src/Writer.vala

```diff
@@ -110,7 +110,7 @@ namespace Writer {
             File? file = null;
 
             do {
-                file_name = "Untitled Document %i".printf (id++);
+                file_name = _("Untitled Document") + "%i".printf (id++);
```

Comment:
```suggestion
                file_name = _("Untitled Document %i").printf (id++);
```

I don't know specifically what language has this construction, but I suppose some language may need to bring numbers at the beginning of translation. Including the string `%s` (called "string format specifier") into translations allows translators to bring the number to any position of their translation.

Reply:
Yes you're completely right!